src/app/common/header/header.tpl.html

```diff
@@ -30,12 +30,12 @@ <h1 class="logo">Doubtfire</h1>
         <ul class="dropdown-menu">
           <li ng-hide="unitRoles.length == 0" class="dropdown-header">Units You Teach</li>
           <li ng-repeat="unitRole in unitRoles | filter:isUniqueUnitRole">
-            <a href="#/units?unitRole={{unitRole.id}}">{{unitRole.unit_name}}</a>
+            <a href="#/units?unitRole={{unitRole.id}}">{{unitRole.unit_code}} - {{unitRole.unit_name}}</a>
           </li>
           <li ng-hide="unitRoles.length == 0 || projects.length == 0" class="divider"></li>
           <li ng-hide="projects.length == 0" class="dropdown-header" ng-hide="{{projects.length}}">Units You Study</li>
           <li ng-repeat="project in projects">
-            <a href="#/projects/{{project.project_id}}">{{project.unit_name}}</a>
+            <a href="#/projects/{{project.project_id}}">{{project.unit_code}} - {{project.unit_name}}</a>
```

Comment:
@Reubsinit Can we add this inside a label to make it pretty?

```html
<label class="label label-info">{{project.unit-code}}</label> {{project.unit_name}}
```

Also I think @macite want's the date too? Was that right?